shared/dataRepository.ts

```diff
@@ -39,3 +47,9 @@ const generateFilePath = (time: NzTimeIso) => {
   const date = DateTime.fromISO(time).toFormat("yyyy-LL-dd");
   return `${CONSTANTS.dataFolder}/${date}.txt`;
 };
+
+const getLatestVaccineDataFilePath = async () => {
+  const files = await fs.readdir(CONSTANTS.dataFolder);
+  const latestDataFile = files.sort()[files.length - 1];
```

Comment:
```suggestion
  const [latestDataFile] = files.sort().reverse();
```

Reply:
> const [latestDataFile] = files.sort().reverse();

Is this common in JS land? Seems harder to understand IMO. 
Happy to change if this is usual pattern.

Reply:
Yeh I think `at` is still proposal stage 3 - but some time soon hoepfully!

I can't comment on whether it's more common, it's probably preference - I like the destructuring version because to me it reads like:

```
const [latestDataFile] // take the first file out of a list of files sorted latest first
    = files.sort().reverse(); // sort files so that latest is first first
```

So another way of writing it might be:
```suggestion
  const sortFilesByName = f => [...f].sort().reverse();
  const [latestDataFile] = sortFilesByName(files);
```

No need to change tho, I'll approve first 🙏

Reply:
Yea I might suggest `[files.length - 1];` is a bit more universal than destructing and keep it as it is :P